rust-runtime/aws-smithy-runtime-api/src/client/http.rs

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! HTTP clients and connectors
+//!
+//! # What is a connector?
+//!
+//! When we talk about connectors, we are referring to the [`HttpConnector`] trait, and implementations of
+//! that trait. This trait simply takes a HTTP request, and returns a future with the response for that
+//! request.
+//!
+//! This is slightly different from what a connector is in other libraries such as
+//! [`hyper`](https://crates.io/crates/hyper). In hyper 0.x, the connector is a
+//! [`tower`](https://crates.io/crates/tower) `Service` that takes a `Uri` and returns
```

Comment:
consider using footnote style links to make this easier to read as raw markdown.

```md
[`hyper`]

// and then later

[hyper]: https://crates.io/crates/hyper
```

I'm pretty sure that works even with the backticks.

Reply:
I didn't realize that worked in rustdoc. That's awesome!